db/migrate/20201202074942_add_address_index_to_patient.rb

```diff
@@ -0,0 +1,7 @@
+class AddAddressIndexToPatient < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :patients, :address_id, algorithm: :concurrently
```

Comment:
We need this to speed up deletes. Deletes check for foreign-keys on other tables and the [EXPLAIN](https://www.postgresql.org/message-id/49f698a3-e4f3-d4e4-f769-d29b39bbb735%40gmail.com) plans don't necessarily show that they are slow because of this.